aws/table_aws_wafv2_web_acl.go

```diff
@@ -74,6 +74,13 @@ func tableAwsWafv2WebAcl(_ context.Context) *plugin.Table {
 				Type:        proto.ColumnType_BOOL,
 				Hydrate:     getAwsWafv2WebAcl,
 			},
+			{
+				Name:        "associated_resources_arns",
```

Comment:
```suggestion
				Name:        "associated_resources",
```

I think the wording can be a bit awkward when trying to include the word `arns`, e.g., `associated_resource_arns` feels a bit off since `resource` is singular, but there can be multiple resources, while `associated_resources_arns` doesn't read well either since you can't indicates `resources` should be posessive.

I think shortening to `associated_resources` is OK, and matches a bit closer to the API name also.